diff --git a/facebook_pull.module b/facebook_pull.module
index 37cd355..a6e3654 100644
--- a/facebook_pull.module
+++ b/facebook_pull.module
@@ -57,7 +57,7 @@ function facebook_pull_menu() {
     'description' => 'Set Facebook API details for facebook blocks.',
     'page callback' => 'drupal_get_form',
     'page arguments' => array('facebook_pull_settings'),
-    'access arguments' => array('administer facebook pull '),
+    'access arguments' => array('administer facebook pull'),
     'type' => MENU_NORMAL_ITEM,
   );
 
